packages/react-core/src/filters/tileFeatures.js

```diff
@@ -188,10 +194,13 @@ export function tileFeatures({ tiles, viewport, geometry, uniqueIdProperty }) {
     if (!clippedGeometryToIntersect) {
       continue;
     }
-    // Transform the geometry to intersect to tile coordinates [0..1]
+    // Transform the geometry to intersect to tile coordinates [0..1] if the tileFormat is MVT
```

Comment:
We could probably add a comment with our assumptions here, in case they change in the future, eg (feel free to rewrite):

We assume that MVT tileFormat uses local coordinates, while in the case of 'geojson', the geometries are already in WGS84

Reply:
Done